readme.md

```diff
@@ -455,9 +455,10 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Userscripts](https://github.com/brunocvcunha/awesome-userscripts) - Enhance your browsing experience.
 - [Pokémon](https://github.com/tobiasbueschel/awesome-pokemon) - Pokémon and Pokémon GO resources and tools.
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
+- [Quantified Self](https://github.com/willempienaar/awesome-quantified-self)
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
-
+- [Quantified Self](https://github.com/willempienaar/awesome-quantified-self) - Quantified Self resources, apps, devices, and more.
```

Comment:
It should be a succinct description of what quantified self is, not what the list contains.

You also have duplicated the entry.